packages/integration/spec/testing/local-dependencies.spec.js

```diff
@@ -0,0 +1,287 @@
+            expect(failure).toBe(false);
+            expect(result).toEqual('hello');
+          });
+        },
+        60 * 1000 * 10
+      );
+    });
+    describe(`In a zeta push case`, () => {
+      it(
+        `Should be able to launch a local worker and call hello CCS`,
+        async () => {
+          await given()
+            /**/ .credentials()
+            /*   */ .fromEnv()
+            /*   */ .and()
+            /**/ .project()
+            /*   */ .template()
+            /*     */ .sourceDir(basicWorkerDir)
+            /*     */ .and()
+            /*   */ .and()
+            /**/ .worker()
+            /*   */ .pushed()
+            /*   */ .and()
+            /**/ .npm()
+            /*   */ .dependencies()
+            /*     */ .module('fake-npm-lib')
+            /*       */ .useSources('./spec/templates/fake-npm-lib')
+            /*         */ .and()
+            /*       */ .and()
+            /*     */ .module('@zetapush/core')
+            /*       */ .useSources('../core')
+            /*         */ .and()
+            /*       */ .and()
+            /*     */ .and()
+            /*   */ .and()
+            /**/ .apply(this);
+
+          await frontAction('2) call hello', this, async (api) => {
+            let failure = false;
+            let result = null;
+            try {
+              result = await api.hello();
+            } catch (error) {
+              failure = true;
+            }
+            expect(failure).toBe(false);
+            expect(result).toEqual('hello');
+          });
+        },
+        60 * 1000 * 10
+      );
+    });
+  });
+
+  describe(`Using modules with modules() syntax`, () => {
+    const myLocalModules = modules()
+      /*                 */ .module('fake-npm-lib')
+      /*                   */ .useSources('./spec/templates/fake-npm-lib')
+      /*                     */ .and()
+      /*                   */ .and()
+      /*                 */ .module('@zetapush/core')
+      /*                   */ .useSources('../core')
+      /*                     */ .and()
+      /*                   */ .and()
+      /*                 */ .apply();
+
+    describe(`In a local case`, () => {
+      it(
+        `Should be able to launch a local worker and call hello CCS`,
+        async () => {
+          await given()
+            /**/ .credentials()
+            /*   */ .fromEnv()
+            /*   */ .and()
+            /**/ .project()
+            /*   */ .template()
+            /*     */ .sourceDir(basicWorkerDir)
+            /*     */ .and()
+            /*   */ .and()
+            /**/ .worker()
+            /*   */ .up()
+            /*   */ .and()
+            /**/ .npm()
+            /*   */ .dependencies()
+            /*     */ .modules(myLocalModules)
+            /*     */ .and()
+            /*   */ .and()
+            /**/ .apply(this);
+
+          await frontAction('call hello', this, async (api) => {
+            let failure = false;
```

Comment:
Code could be simplfied:
```javascript
try {
  const result = await api.hello();
  expect(result).toEqual('hello');
} catch(error) {
   fail('Should not have failed');
}
```
this.context.runner.stop() is not necessary here and is already done automatically by autoclean

Reply:
Done